### ISSUE TITLE
Delight: Update the DM empty state dialog

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-web#22065]
### Your use case

<img width="764" alt="Screenshot 2022-05-04 at 15 13 03" src="https://user-images.githubusercontent.com/89144281/166834177-b2fc99b7-ef23-4c3d-a801-cb0fd5139157.png">

**Comment by t3chguy** [comment]
Isn't this dialog going away with Spotlight?

**Comment by daniellekirkwood** [comment]
It is? What's the new design? 

@niquewoodhouse & @kittykat

**Comment by niquewoodhouse** [comment]
I don't think it is but maybe I've assumed something incorrectly @kittykat @gsouquet?

**Comment by t3chguy** [comment]
Isn't spotlight going to feature ways to search for public rooms & users? If it is why maintain an entirely different dialog also to search for users?

**Comment by gsouquet** [comment]
Yes it is.
I believe the idea is to consolidate the "start DM", public room directory and local filtering under the same umbrella

It does not need to be transitioned immediately when we move spotlight out of beta, but that's the ultimate goal I suppose. @niquewoodhouse let me know if you think that would change some of the design decisions we've made in that area

**Comment by niquewoodhouse** [comment]
>  let me know if you think that would change some of the design decisions we've made in that area

I understand if that's the ultimate goal _maybe_ but, in the immediate future, it can't replace start DM as it doesn't have functionality, like:

- Pick multiple people to DM
- Invite via email to start DM

Search is looking for someone, but starting a DM might be wanting to share a link, inviting via email etc

<img width="612" alt="image" src="https://user-images.githubusercontent.com/1587166/167129168-dcd318f7-a110-4acf-8839-f5b8e22f5659.png">

**Comment by kittykat** [comment]
Update: after discussion, we decided to link to this dialog from the new search experience when people want to create new group chats. (Germain will file additional issues around new user experiences with creating group chats.)

**Comment by daniellekirkwood** [comment]
In that case I'm assigning this to @gsouquet :) 

I will ask y'all to run me thru this next week but from what I can gather it sounds like it would fit right in to new user journey work so @gsouquet please tag the new issues with Z-NewUserJourney

**Comment by gsouquet** [comment]
I believe keeping this issue as the main reference will fit the bill both for the the new users journey, and also to help getting the new search experience out of beta (as this create DM dialog, will be the main entry point to create multi-party DMs)

@niquewoodhouse would you be able to share some a redesign of this direct message modal, you've shared [some early thoughts](https://github.com/vector-im/element-web/issues/22065#issuecomment-1119552575), but it'd be good to have something a bit more formalised

After scanning the code a bit more, here are some things we need to consider: 

- This dialog can not display suggestions if no identity server is set. There's a screen to incentivise people to add one, but we might want to make it more appealing 
- Explain that we can invite multiple people

**Comment by daniellekirkwood** [comment]
While we're morphing this issue into the Search solution (and then using the same modal in both places) can we check with the folks looking at the Start a DM flow to make sure we're not missing anything they might know about? (I think they're focussing on mobile only but not sure). @niquewoodhouse the Figma I have from G is here: https://www.figma.com/file/OvsSONqAyabcUDYp8tnEs6/Start-a-DM?node-id=68%3A31998

**Comment by daniellekirkwood** [comment]
Current state as discussed in out meeting yesterday: 

## Is this issue needed in order to bring search out of Beta?

I believe the answer to be no, WDYT @kittykat 

If the answer is no, then we'll push this back to the Delight backlog and work on other things first.

**Comment by kittykat** [comment]
@daniellekirkwood this isn't needed to bring search out of beta. The epic for exit out of beta is up to date 👍

**Comment by daniellekirkwood** [comment]
Great, removing Janne from the Assignees and moving to the Next column as we'll pick this back up after Search & Welcome home work
